addons/stock/report/picking_templates.xml

```diff
@@ -62,8 +62,8 @@
                                                 <t t-if="move_line.product_id.tracking != 'none'">
                                                     <tr>
                                                         <td class="text-center align-middle">
-                                                            <t t-if="move_line.lot_name or move_line.lot_id">
-                                                                <img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' % ('Code128', move_line.lot_name, 600, 150)" style="width:100%;height:4rem" alt="Barcode"/>
+                                                            <t t-if="move_line.lot_name or move_line.lot_id">                        
+                                                                <div t-field="move_line.lot_name or move_line.lot_id.name" t-options="{'widget': 'barcode', 'symbology': 'Code128', 'width': 600, 'height': 150, 'img_style': 'width:100%;height:4rem', 'img_alt': 'Barcode'}"/>
```

Comment:
an expression in a t-field ?
t-esc instead probably

Reply:
the OR : not the same behavior that previously but probably a fix

Reply:
todo: check rendering on runnign instance

Reply:
A bad copy paste